sdk/connectedvmwarevsphere/Azure.ResourceManager.ConnectedVMwarevSphere/tests/tests/Helpers/ConnectedVMwareTestBase.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Resources.Models;
+using Azure.ResourceManager.TestFramework;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+using Azure.Core;
+using System.Linq;
+
+namespace Azure.ResourceManager.ConnectedVMwarevSphere.Tests.Helpers
+{
+    [ClientTestFixture]
+    public class ConnectedVMwareTestBase : ManagementRecordedTestBase<ConnectedVMwareManagementTestEnvironment>
+    {
+        public const string DefaultLocation = "eastus";
+        protected ArmClient Client { get; private set; }
+        protected Subscription DefaultSubscription { get; private set; }
+        public const string EXTENDED_LOCATION_TYPE = "CustomLocation";
+        public const string CustomLocationId  = "/subscriptions/204898ee-cd13-4332-b9d4-55ca5c25496d/resourceGroups/service-sdk-test/providers/Microsoft.ExtendedLocation/customLocations/service-sdk-test-cl";
```

Comment:
Shall we extract the subscription ID `204898ee-cd13-4332-b9d4-55ca5c25496d` here?

Reply:
In future may be we will only give the customLocation name not the full id and we will generate the full id of custom location through other function. So it is better to not extract from here and used from set env variable.